plugins/woocommerce-blocks/assets/js/extensions/shipping-methods/pickup-location/utils.ts

```diff
@@ -54,9 +54,10 @@ export const getInitialSettings = (): ShippingMethodSettings => {
 	const settings = hydratedScreenSettings.pickupLocationSettings;
 
 	return {
-		enabled: settings?.enabled
-			? settings?.enabled === 'yes'
-			: defaultSettings.enabled,
+		enabled:
+			typeof settings?.enabled !== 'undefined'
```

Comment:
Given that this is a boolean, couldn't we check against that type?

```suggestion
			typeof settings?.enabled == 'boolean'
```

Reply:
Great suggestion, thanks Niels